add section re updates channels on sle (#466)
uyuni-project/uyuni-docs: modules/client-configuration/pages/channels.adoc

If you need to transform Appstream repositories into non-modular repositories for use with some upgrade operations, you can use the Appstream filter with the content lifecycle management tool.
For more information about the Appstream filter, see xref:administration:content-lifecycle-examples.adoc[].

> == Updates Channels on {sle} Clients

Reviewer: There is an important difference between the normal update channel and the "installer update channel".

Update channel exist for all products.

New are the Installer Updates Channel which contain only updates for the  OS "installer" (yast, libzypp, curl, etc).
These channels require in SCC no credentials and are used to update the unified installer media before it installes the OS.
All packages which are in Installer Updates Channel are also in the normal Update Channel. But the normal Update Channel can only be used after the product is registered. Installer Update Channels can be used also before registration and therefore can also fix bugs in the registration code itself.

We need to make it clear. This section is for the Installer Updates.

Author: ah ha, that makes sense. i'll edit.

If you need to transform Appstream repositories into non-modular repositories for use with some upgrade operations, you can use the Appstream filter with the content lifecycle management tool.
For more information about the Appstream filter, see xref:administration:content-lifecycle-examples.adoc[].

== Unified Installer Updates Channels on {sle} Clients

> All {sle} clients require access to an installer updates channel.
> This channel is used by the unified installer to ensure it is up to date before it installs the operating system.

Reviewer: ```suggestion
This channel is used by the unified installer to ensure it is up to date before it installs the operating system.
All {sle} proucts should have access to the installer updates channel during installation.
```

Author: Why reverse these sentences?

Reviewer: The registered client does not need this channel. So I think it is more important to say that the unified installer need it for installing a client.
But more important is the change that the client does not need access to that channel, but all products should get it. If this is given, the rest should work out of the box.
You can change the order, but the change of the first sentence is important from my point of view. Otherwise we give a wrong impression.

Author: OK, updated.

> For {sles} clients the installer updates channel is installed by default when you add a product that contains them, and are enabled when you first synchronize the client.

Reviewer: ```suggestion
For {sles} clients the installer updates channel is synced by default when you add a product that contains them, and are enabled when you create an autoinstallation distribution with these product channels.
```

For all other {sle} variants, including {sle} for SAP,  you must add the installer updates channel manually.

> To do this, clone the installer updates channel that belongs to the appropriate base channel, and assign it to the client.

Reviewer: ```suggestion
To do this, clone the appropriate {sles} installer updates channel below the base channel of these {sle} variants.
When creating a autoinstallation distribution for these {sle} variants after the channel was cloned , it will be used automatically.
```

If you need to transform Appstream repositories into non-modular repositories for use with some upgrade operations, you can use the Appstream filter with the content lifecycle management tool.
For more information about the Appstream filter, see xref:administration:content-lifecycle-examples.adoc[].

== Unified Installer Updates Channels on {sle} Clients

This channel is used by the unified installer to ensure it is up to date before it installs the operating system.
All {sle} proucts should have access to the installer updates channel during installation.

For {sles} clients the installer updates channel is synchronized by default when you add a product that contains them, and are enabled when you create an autoinstallation distribution with these product channels.

For all other {sle} variants, including {sle} for SAP,  you must add the installer updates channel manually.
To do this, clone the appropriate {sles} installer updates channel below the base channel of these {sle} variants.

> When creating a autoinstallation distribution for these {sle} variants after the channel was cloned , it will be used automatically.

Reviewer 2: ```suggestion
When creating a autoinstallation distribution for these {sle} variants after the channel was cloned, it will be used automatically.
```
